api/src/index.ts

```diff
@@ -1,39 +1,58 @@
+require("dotenv").config();
 import Fastify from "fastify";
 import fetch from "node-fetch";
 import { generateNbTxsPerDay } from "./tasks/generateNbTxsPerDay";
 import { generateUniqueAddressGrowingPerDay } from "./tasks/generateUniqueAddressGrowingPerDay";
 import { generateTxsFeePerDay } from "./tasks/generateTxsFeePerDay";
 import { readData, writeData, FileData } from "./utils";
+import {
+  tweetStatsQueue,
+  tweetStatsQueueScheduler,
+} from "../src/twitterBot/bullQueue";
 
 let cacheData: FileData | false = false;
 async function generateDataStats() {
   const currentData = readData();
-
   console.log("Starting number of transactions...");
   const nbTxsPerDay = await generateNbTxsPerDay(currentData?.nbTxsPerDay);
   console.log("Number of transactions generated");
-
   console.log("Starting number of unique addresses...");
   const uniqueAddressGrowingPerDay = await generateUniqueAddressGrowingPerDay();
   console.log("Number of unique addresses generated");
-
   console.log("Starting total fees...");
   const txsFeePerDay = await generateTxsFeePerDay(currentData?.txsFeePerDay);
   console.log("Total fees generated");
-
   const fileData = {
     nbTxsPerDay,
     uniqueAddressGrowingPerDay,
     txsFeePerDay,
   };
-
   writeData(fileData);
   cacheData = fileData;
 }
 
 generateDataStats()
-  .then(() => {
+  .then(async () => {
     console.log("First data generated");
+
+    ({
+      work: await tweetStatsQueue.removeRepeatableByKey(
```

Comment:
@Akirtovskis is there a way to delete all of them directly?

Reply:
Otherwise if we change the hour at some point it will not delete the old hour

Reply:
Didn't find a real way to remove all of the directly, so would suggest that for now we go with this. In case of change a simple fix is just to temporary add one more `removeRepeatableByKey` fn with the old values.